src/e2e-tests/top-up-create-stream.e2e.test.ts

```diff
@@ -168,7 +168,6 @@ describe('app', async () => {
       it('displays the original users outgoing stream on their profile', async () => {
         await page.goto('http://127.0.0.1:3000/app/0x433220a86126eFe2b8C98a723E73eBAd2D0CbaDc');
 
-        await expect(page.locator('text=↑ Outgoing')).toHaveCount(1);
```

Comment:
Removing this kinda removes the entire check that this new stream is being interpreted by the streams store as outgoing, rather than incoming. 😬 Would be good to keep that

Reply:
maybe just check that _neither_ "incoming" or "outgoing" appear at this point. that should do it

Reply:
In 3f2ed36, I check for "From" and "Total streamed" since outgoing/incoming = 0 would be true in any situation without both stream types present

Reply:
g2g?